roles/openshift_node_group/templates/node-config.yaml.j2

```diff
@@ -50,8 +50,7 @@ kubeletArguments:
   cloud-provider:
   - {{ openshift_node_group_cloud_provider }}
 {% endif %}
-  node-labels: 
-  - "{{ openshift_node_group_labels | join(',') }}"
+  node-labels: []
```

Comment:
I think this is a good place for a loop or | to_padded_yaml filter.

Reply:
see comment above about labels not working if passed in and an existing cm exists.

Reply:
If the config map already exists, this template shouldn't be used?

Reply:
Yes, that is the logic that was pre-existing in this playbook.  I'm expanding this logic to include the newly included volume-config that was added last week.

Reply:
and for the record, I believe this is the appropriate behaviour